stage1/init/kvm.go

```diff
@@ -30,8 +35,12 @@ import (
 	"github.com/hashicorp/errwrap"
 )
 
-// KvmPodToSystemd generates systemd unit files for a pod according to the manifest and network configuration
-func KvmPodToSystemd(p *stage1commontypes.Pod, n *networking.Networking) error {
+const (
+	sharedVolPerm = os.FileMode(0755)
```

Comment:
Hmmm... This would require to export (uppercase) this constant in common/pod.go
I'm doing this now.